src/bci_build/package/__init__.py

```diff
@@ -1320,9 +1344,22 @@ async def write_file_to_dest(fname: str, contents: str | bytes) -> None:
                 False
             ), f"got an unexpected build_recipe_type: '{self.build_recipe_type}'"
 
+        if self.build_flavor:
+            flavors: str = "\n".join(
+                " " * 4 + f"<package>{pkg}</package>"
```

Comment:
This looks really odd, just to produce what this does:

```suggestion
                f"    <package>{pkg}</package>"
```

Reply:
I guess this comes down to a personal preference and DRY, don't care either way.